### PR TITLE
chore:cut v0.73.2

### PR DESCRIPTION
**Pull request by slashpai**
## Description

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue._

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Verification

Please check the [Prometheus-Operator testing guidelines](../TESTING.md) for recommendations about automated tests.

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

**Comment by simonpasquier** [comment]
can we get https://github.com/prometheus-operator/prometheus-operator/pull/6525 too?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.73.2 / 2024-04-18
+
+* [BUGFIX] Fix ScrapeClassTLSConfig nil pointer exception #6526
```
Comment (simonpasquier) [inline CHANGELOG.md:3]:
```suggestion
* [BUGFIX] Fix ScrapeClassTLSConfig nil pointer exception #6526
* [BUGFIX] Fix missing `prometheus_operator_kubernetes_client_http_requests_total` metric. #6525 
``

**Comment by slashpai** [comment]
Should we wait till https://github.com/prometheus-operator/prometheus-operator/issues/6523 is fixed?

**Review by kakkoyun — APPROVED** [review]

**Comment by simonpasquier** [comment]
I commented on https://github.com/prometheus-operator/prometheus-operator/issues/6523 and i think that it's best to not include it for now.

**Comment by slashpai** [comment]
@simonpasquier please review again fixed generate issue

**Comment by simonpasquier** [comment]
:ship:

**Pull request merged**